models/articles-models.js

```diff
@@ -0,0 +1,34 @@
+const db = require("../db/connection");
+
+exports.selectArticlesById = (id) => {
+  return db
+    .query(
+      "SELECT articles.*, COUNT (comments.article_id)::INTEGER AS comment_count FROM comments RIGHT JOIN articles ON comments.article_id = articles.article_id WHERE articles.article_id=$1 GROUP BY articles.article_id;",
```

Comment:
You can change this before your next PR if you like, but if you us `` instead of "" here, you can split your query statements out onto multiple lines which will make it way more readable.